test/unit/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "plugin:jasmine/recommended",
+  "plugins": [
+    "jasmine"
+  ],
+  "env": {
+    "node": true,
+    "jasmine": true,
+    "es6": true
+  },
+  "rules": {
+    "max-len": 0,
+    "complexity": 0,
+    "max-params": 0,
+    "max-statements": 0,
```

Comment:
Hmm. We should revisit these rules at some point. Do you recall why these were disabled for tests?

Reply:
tests tend to get many statements hence just disable it globally instead of case by case.